src/py/flwr/client/app.py

```diff
@@ -90,7 +92,8 @@ def start_client(
     client: Client,
     grpc_max_message_length: int = GRPC_MAX_MESSAGE_LENGTH,
     root_certificates: Optional[Union[bytes, str]] = None,
-    rest: bool = False,
+    rest: bool = False,  # Deprecated in favor of `transport`
```

Comment:
As this is experimental anyway should we just remove it?

Reply:
I'd propose to keep it around for just one release. The maintenance overhead is minimal, and it will make the transition easier.